docs/jobs/ai-assistant.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Using OpenFn with AI Assistant
+sidebar_label: Using AI Assistant
+---
+
+:::warning
+
+AI makes mistakes. Always review the AI's response carefully and make sure to
+verify the output. You can help us improve the AI Assistant by providing
+feedback on the response.
+
+:::
+
+## About OpenFn AI Assistant
+
+The AI Assistant is built on the Claude 3.5 Sonnet model from Anthropic and is
+trained with OpenFn documentation and example job code. The goal of the
+assistant is to help you with your jobs and diagnosing issues with your jobs and
+error logs. It is important to note that the assistant is not a replacement for
+human developers. It is a tool to help you with your jobs and diagnosing issues
+with your jobs and error logs.
+
+:::caution
+
+Before you start, we recommend that you take a moment to read our more about the
+AI Assistant and
+[Responsible AI principles](https://www.openfn.org/responsible-ai/) by clicking
+on the `Learn more about AI Assistant` above the button.
+
+:::
+
+## How to access the AI Assistant
+
+The AI Assistant is located in the IDE/Inspector as a new tab in the Input
```

Comment:
Is it the IDE or Inspector?

I think we should mention that, for now at least, the inspector is only available in the job editor (users probably don't know what the inspector is).

The assistant is pretty discoverable tbh, I don't think we need heavy docs here

Reply:
We call it the `Inspector` throughout the existing docs fyi @christad92